packages/backend/src/config/db.ts

```diff
@@ -0,0 +1,16 @@
+import dotenv from "dotenv"
+
+dotenv.config()
+const MONGODB_URL = String(process.env.MONGODB_URL)
+const SERVER_PORT = process.env.SERVER_PORT
```

Comment:
Why do you create it here? You have SERVER_PORT in app.ts.

Reply:
du har rätt, radera const port i app.ts